website/SceneLinear.rst

```diff
@@ -238,8 +244,14 @@ OpenColorIO.
 
 .. rubric:: Footnotes
 
+.. [#fterms] Color scientists use a bewilderingly large number of special terms and
+   acronyms. Some use two different terms and mean exactly the same thing; others
+   might insist there is a subtle but important distinction between them. To keep things brief,
```

Comment:
LOL! I have nothing to suggest here, just noting that you made me snort tea through my nose